account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -83,7 +80,7 @@ def filter_lines(self, lines):
         """
         self.ensure_one()
         payment_lines = self.env['payment.line'].\
-            search([('order_id.state', 'in', ('draft', 'open')),
+            search([('order_id.state', '!=', 'cancel'),
                     ('move_line_id', 'in', lines.ids)])
```

Comment:
The return would not work for us, as we import the returns from a bank statement. We would thus don't need to generate other move lines.
As @sbidoul noticed, I think we should try to get this working without the need of any other module. This is why I proposed to add the ability of removing an invoice from a payment order (=> moving it to a cancelled payment order).